src/UglyToad.PdfPig/Geometry/GeometryExtensions.cs

```diff
@@ -11,6 +11,19 @@
     /// </summary>
     public static class GeometryExtensions
     {
+        private const double epsilon = 1e-5;
+
+        /// <summary>
+        /// Return true if the points are in counter-clockwise order.
+        /// </summary>
+        /// <param name="point1">The first point.</param>
+        /// <param name="point2">The second point.</param>
+        /// <param name="point3">The third point.</param>
+        private static bool ccw(PdfPoint point1, PdfPoint point2, PdfPoint point3)
```

Comment:
A better name for this would be `IsCounterClockwise`.

Reply:
I would leave it like this as it is the usual way this function is called in computational geometry.